components/raftstore/src/store/peer.rs

```diff
@@ -2319,7 +2319,11 @@ where
                 if self.check_disk_usages_before_propose(ctx, disk_full_opt, &mut stores) {
                     self.propose_normal(ctx, req)
                 } else {
-                    let errmsg = String::from("propose failed: disk full");
+                    // let errmsg = String::from("propose failed: disk full");
+                    let errmsg = format!(
+                        "propose failed: disk full, cmd::disk_full_opt={:?}, diskUsage={:?}",
```

Comment:
Maybe it's not caused by `self_disk_usage`. It's better to show disk full stores instead.

Reply:
It's caused by the judge condition, showed at check_disk_usages_before_propose, as what the changes describe.